### ISSUE TITLE
Color Picker support for Stylus

### ISSUE THREAD

**Issue by octref** [vuejs/vetur#856]
Color Picker support for Stylus.

This would be `onDocumentColor` for Stylus, similar to https://github.com/Microsoft/vscode-css-languageservice/blob/47e6f7e4d09de7142f5275658037eda39008b661/src/services/cssNavigation.ts#L180-L187

@ThisIsManta Would you have time to take a look?

**Comment by ThisIsManta** [comment]
I'm not familiar with the language server APIs. Hence, I would not give you any promises, but I will take a look on this on weekends.

Let me know if you have people handling this, so I can step away.

**Comment by kevinmarrec** [comment]
@octref  @ThisIsManta 
Is it related to this issue ? (Color preview not displayed if lang="stylus")

![screenshot_3](https://user-images.githubusercontent.com/25272043/48161213-4d27c500-e2da-11e8-95c0-a1421327791d.png)

If not, please tell me, I'll open a new one.

**Comment by octref** [comment]
Yes, Stylus does not have `documentColor` support so it does not show the color preview.
